server/controllers/assessmentDayDataController/index.js

```diff
@@ -0,0 +1,66 @@
+import AssessmentDayDataModel from '../../models/AssessmentDayDataModel'
+import DataFileModel from '../../models/DataFileModel'
+import AssessDayDataService from '../../services/AssessmentDayDataService'
+import _ from 'lodash'
+
+const AssessmentDayDataController = {
+  create: async (req, res) => {
+    try {
+      if (_.isEmpty(req.body))
+        return res.status(400).json({ message: 'The request is empty' })
+
+      const { appDb } = req.app.locals
+      const newAssessmentData = []
+      const updatedData = []
+      const assessmentDataService = new AssessDayDataService()
+
+      for await (const { metadata, subject_assessments } of req.body) {
+        if (metadata && subject_assessments.length) {
+          const { assessment, subject } = metadata
+          const query = {
+            assessment,
+            subject,
+          }
+          const storedAssessmentData = await AssessmentDayDataModel.all(
+            appDb,
+            query
```

Comment:
This query object contains arbitrary JSON inputs from the user, and leaves us exposed to the same mongo injection issues we see elsewhere in the app. For a POC I don't consider this a blocker, I just want to raise it so we make sure to correct things when we have prioritization from the client.